packages/clima_ui/lib/widgets/weather_widget.dart

```diff
@@ -20,98 +20,137 @@ class WeatherWidget extends HookWidget {
       return const SizedBox.shrink();
     }
 
+    Widget emptySpace() {
+      if (MediaQuery.of(context).size.aspectRatio < 0.4736842105) {
+        return const Spacer(
+          flex: 2,
+        );
+      } else if (MediaQuery.of(context).size.aspectRatio < 0.5294117647) {
+        return const Spacer();
```

Comment:
Please use explicit ratios instead of these magic numbers. For example, `0.5625` would be `9.0 / 16.0` (ratio of width to height).